.gitignore

```diff
@@ -2,4 +2,3 @@ data/*.sql
 data/*.sql.gz
 config/portal.properties
 **/.DS_Store
-.env
```

Comment:
We explicitly removed the .env file at the project root in favor of a .env.example file that, so that the .env is not under version control. This avoids merge conflicts when local adjustments to the .env file are made (eg. inserting OncoKB Token).

However, for you use case I would to follow either the same pattern or explicitly whitelist your .env file using `!tools/cbpManager.deploy/.env`